src/styles/screenreaderOnly.ts

```diff
@@ -0,0 +1,13 @@
+import { CSSProperties } from 'react';
+
+export const screenreaderOnlyStyles: CSSProperties = {
+  position: 'absolute',
+  width: '1px',
+  height: '1px',
+  padding: 0,
+  margin: '-1px',
+  overflow: 'hidden',
+  clip: 'rect(0, 0, 0, 0)',
+  whiteSpace: 'nowrap',
+  borderWidth: 0,
+};
```

Comment:
Took this from https://tailwindcss.com/docs/screen-readers#class-reference - works well!